### PR TITLE
Update app to v3.28 of ember

### PR DESCRIPTION
**Pull request by MehulKChaudhari**
This PR updates this app to the v3.28 version of ember. 

Issue: #224

### CODE REVIEW

**Comment by MehulKChaudhari** [comment]
Hey @jenweber, I have updated this repo to v3.28. Do I have to update this to v4 in the same PR?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+branches:
+  only:
+    - master
+
+script:
+  - npm test
```
Comment (ijlee2) [inline .travis.yml:25]: This project uses GitHub Actions for continuous integration. I forgot to mention, after letting Ember CLI run, you will want to delete `.travis.yml`.

**Comment by ijlee2** [comment]
The lint checks failed:

```sh
/home/runner/work/ember-help-wanted/ember-help-wanted/tests/.eslintrc.js
Error:   1:1  error  'module' is not defined  no-undef
```

This is strange to me because a new Ember app doesn't create the file `tests/.eslintrc.js`. Maybe it had once done, based on its [commit history](https://github.com/ember-learn/ember-help-wanted/commits/main/tests/.eslintrc.js) (Ember v2.13)?

Let's delete the file and check if it helps CI pass.

**Review by ijlee2 — CHANGES_REQUESTED** [review]
It looked to me like you were able to run `ember-cli-update` without a problem. 💯

Before we merge the pull request, there are 2 things I'd like you to do:

- Remove `travis.yml`
- Remove `tests/.eslintrc.js`

And yes, let's create a separate pull request for updating `ember-source` from 3.28 to 4.3.

I think you will run into a merge conflict because `ember-cli-update` will introduce the default `ci.yml` file for GitHub Actions. Please ensure that the current YAML file (https://github.com/ember-learn/ember-help-wanted/blob/main/.github/workflows/ci.yml) remains unchanged in your pull request.

File: .travis.yml (hunk 1 above)
Comment (MehulKChaudhari) [reply .travis.yml:25]: ok I will delete it.

**Comment by MehulKChaudhari** [comment]
@ijlee2 Made changes as per your suggestions.

**Review by ijlee2 — APPROVED** [review]
Thanks for your first contribution to `ember-help-wanted`!

**Comment by MehulKChaudhari** [comment]
@ijlee2 now I should create one more PR to upgrade the app to v4?

**Comment by ijlee2** [comment]
@MehulKChaudhari Yep.

First, you will want to check that you have the latest code in your local `main` branch. GitHub provides you a way to "fetch upstream" (upstream refers to the original repo, `https://github.com/ember-learn/ember-help-wanted`) so that you don't need to use the terminal and remember the related commands.

In your [forked repo](https://github.com/MehulKChaudhari/ember-help-wanted), look for the button `Fetch upstream`:

fetch_upstream

Once you fetch the upstream, I think you will see a text like `This branch is up to date with ember-learn/ember-help-wanted:main.`

That's when you, in your terminal, can run `git pull` so that you have the latest code on your local machine.

```sh
# Switch branch to main
git checkout main

# Get the latest code from your forked repo
git pull

# Update dependencies
yarn install

# Create a branch
git checkout -b update-ember-to-4.3
```

Let me know if these steps worked?

**Comment by MehulKChaudhari** [comment]
> @MehulKChaudhari Yep.
> 
> First, you will want to check that you have the latest code in your local `main` branch. GitHub provides you a way to "fetch upstream" (upstream refers to the original repo, `https://github.com/ember-learn/ember-help-wanted`) so that you don't need to use the terminal and remember the related commands.
> 
> In your [forked repo](https://github.com/MehulKChaudhari/ember-help-wanted), look for the button `Fetch upstream`:
> 
> fetch_upstream
> 
> Once you fetch the upstream, I think you will see a text like `This branch is up to date with ember-learn/ember-help-wanted:main.`
> 
> That's when you, in your terminal, can run `git pull` so that you have the latest code on your local machine.
> 
> ```shell
> # Switch branch to main
> git checkout main
> 
> # Get the latest code from your forked repo
> git pull
> 
> # Update dependencies
> yarn install
> 
> # Create a branch
> git checkout -b update-ember-to-4.3
> ```
> 
> Let me know if these steps worked?

Yes, I did all the steps as you suggested.

**Pull request merged**